### PR TITLE
Add test for animated_container.0.dart API example.

### PR DESCRIPTION
**Pull request by ksokolovskyi**
This PR contributes to https://github.com/flutter/flutter/issues/130459

### Description
- Adds test for `examples/api/test/widgets/implicit_animations/animated_container.0_test.dart`

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [X] I signed the [CLA].
- [X] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [X] I added new tests to check the change I am making, or this PR is [test-exempt].
- [X] I followed the [breaking change policy] and added [Data Driven Fixes] where supported.
- [X] All existing and new tests are passing.

### CODE REVIEW

File: examples/api/test/widgets/implicit_animations/animated_container.0_test.dart [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      (container.decoration! as BoxDecoration).color,
+      equals(Colors.blue),
+    );
+    expect(
+      container.constraints,
+      equals(const BoxConstraints.tightFor(width: 100, height: 200)),
+    );
+    expect(container.alignment, equals(Alignment.topCenter));
+
+    await tester.pump(const Duration(seconds: 2));
```
Comment (Renzo-Olivares) [inline examples/api/test/widgets/implicit_animations/animated_container.0_test.dart:51]: nit: maybe add a comment that 2 seconds is the duration of the animation.

**Review by Renzo-Olivares — APPROVED** [review]
LGTM with small nits. Thank you for adding this test!

File: examples/api/test/widgets/implicit_animations/animated_container.0_test.dart (hunk 1 above)
Comment (ksokolovskyi) [reply examples/api/test/widgets/implicit_animations/animated_container.0_test.dart:52]: @Renzo-Olivares, thanks for the review, could you please take a look at the latest commit?

Comment (Renzo-Olivares) [reply examples/api/test/widgets/implicit_animations/animated_container.0_test.dart:52]: Maybe, `// Advance animation to the end by the 2-second duration specified in the example app.` . I just wanted to make it clear where the 2 seconds is coming from.

**Review by Renzo-Olivares — APPROVED** [review]
LGTM

**Comment by ksokolovskyi** [comment]
@Renzo-Olivares thanks for the review!

**Review by gspencergoog — APPROVED** [review]
32384589-a60f0e74-c078-11e7-9bc1-e5b5287aea9d

**Pull request merged**